Controllers/InterventionViewController.swift

```diff
@@ -405,7 +412,7 @@ class InterventionViewController: UIViewController, UITableViewDelegate, UITable
 
   @objc func action(sender: UIButton) {
     hideInterventionAdd()
-    performSegue(withIdentifier: "addIntervention", sender: sender)
+    performSegue(withIdentifier: "ShowAddInterventionVC", sender: sender)
```

Comment:
Segue identifiers should be written in camelCase, with a first letter lowercased